src/App.tsx

```diff
@@ -20,17 +24,22 @@ export const App: React.FC = () => {
                 id="search-query"
                 className="input"
                 placeholder="Type search word"
+                value={query}
+                onChange={(event) => {
+                  setQuery(event.target.value);
+                }}
               />
             </div>
           </div>
         </div>
 
-        <MoviesList movies={moviesFromServer} />
+        <MoviesList
+          movies={visibleMovies}
+          query={query}
+        />
```

Comment:
this MoviesList don't need two parameters
realize filter in this component and pass only one parameter